Apply rebranding for /kubernetes/what-is-kubernetes

britneywwc opened the pull request:
## Done

- Rebrand /kubernetes/what-is-kubernetes

## QA

- Go to https://ubuntu-com-14677.demos.haus/kubernetes/what-is-kubernetes
- Check against [copy doc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit?tab=t.0) and [Figma](https://www.figma.com/design/l4wd5ry1KQ6fHbrWptvQbE/24.04-Kubernetes-bubble?node-id=605-9862&m=dev)

## Issue / Card

Fixes [WD-12891](https://warthogs.atlassian.net/browse/WD-12891)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-12891]: https://warthogs.atlassian.net/browse/WD-12891?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://ubuntu-com-14677.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/12018)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

eliman11 commented:
Thanks @britneywwc! Just a few quick comments:
- [ ] "[Find more resources](https://ubuntu.com/kubernetes/resources)" link is returning a 404 - looks like there's an extra /kubernetes in the URL
- [ ] "[Contact us to get your K8s questions answered today ›](https://ubuntu-com-14677.demos.haus/kubernetes/js-invoke-modal)" also returning a 404 for me
- [ ] Could we change the glossary table into a divided list without changing the styling to help with the accessibility of the page?
- [ ] On smaller screens - have that first element span the full width of the screen
<img width="564" alt="Screenshot 2025-01-24 at 10 22 00" src="https://github.com/user-attachments/assets/a1d7421b-d28b-44cb-8d4d-cfe20d62c3dd" />

mattea-turic commented:
Thanks @britneywwc !

Feedback:

For "What is a Kubernetes cluster?":
- [ ] Could you add a hr above the paragraph section below the list pls? i.e. "Every cluster has at least one worker..."

For "Kubernetes advantages"
- [ ] As @eliman11 mentioned, the hrs aren't extending full-width
- [ ] And for smaller screens, could you use the low-contrast lines pls?

For the logo section "Key contributors to Kubernetes":
- [ ] The logos look sorta small/they seem to be contained within a square rather than just as their variable-width (104px height container)? Is there something that I might've over looked when uploading the logos to the assets manager?

For "What does Kubernetes do?"
- [ ] On medium screens, there are just too many checklist items t_t could we maybe just omit the 50/50 layout here and just stack everything on top of each other full-width?

For "What Kubernetes is not":
- [ ] Same as above pls

For "How does Kubernetes work?":
- [ ] Could you resize the chart container to be cinematic aspect ratio and switch out the current asset for [this one](https://assets.ubuntu.com/v1/104a2311-chart.png)
- [ ] For medium screens, again, could we omit the 50/50 layout pls?

britneywwc commented:
Thanks @eliman11 @mattea-turic, I've addressed both of your comments, could you please have another look?


> For the logo section "Key contributors to Kubernetes":
>  The logos look sorta small/they seem to be contained within a square rather than just as their variable-width (104px height container)? Is there something that I might've over looked when uploading the logos to the assets manager?

@mattea-turic I removed the logo item class on images, the only thing is AWS logo looks slightly bigger and distorts the sizes of other logos. Can you upload another AWS logo pls?

eliman11 commented:
Thanks @britneywwc looks good! Adding +1

mattea-turic commented:
@britneywwc I tried reuploading, but it tells me that there's already the exact same file so it should be the correct size :') I uplaoded [@2x](https://assets.ubuntu.com/manager?tag=what-is-k8s-logos) tho, which might help?
<img width="500" alt="Screenshot 2025-01-28 at 01 03 38" src="https://github.com/user-attachments/assets/f2ecca41-57cd-4192-a02b-79744ed7a670" />

So basically, just two more things to change:
- [ ] Logo section
- [ ] And the chart within the cinematic container still doesn't seem to reflect the way it looks in figma. maybe it's also an assets thing, so i uploaded [another image](https://assets.ubuntu.com/v1/104a2311-chart-2.png) which should hopefully work!

mtruj013 commented on templates/kubernetes/what-is-kubernetes.html:
```diff
@@ -15,914 +18,716 @@
-        </p>
+  <hr class="p-rule is-fixed-width" />
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width u-vertically-center">
```
```suggestion
    <div class="u-fixed-width">
```

mtruj013 reviewed (changes requested):
Thanks @britneywwc ! Some comments in the code:

mattea-turic commented:
@mtruj013 

- [ ] Omg I finally solved the cinematic image issue (for "How does Kubernetes work?"). I realised I had to export the chart with clipping applied so finally it shouldn't be so small :') Image here!

- [ ] And then "Kubernetes glossary" is missing a bottom reg. padding

I'll +1 for design
